test/stats/stats_aggregate_test.cpp

```diff
@@ -235,4 +235,85 @@ OpMemoryModel Logical GLSL450
   EXPECT_EQ(2u, stats.opcode_hist.at(SpvOpExtension));
 }
 
+TEST(AggregateStats, OpcodeMarkovHistogram) {
+  const std::string code1 = R"(
+OpCapability Shader
+OpCapability Linkage
+OpExtension "SPV_NV_viewport_array2"
+OpMemoryModel Logical GLSL450
+)";
+
+  const std::string code2 = R"(
+OpCapability Addresses
+OpCapability Kernel
+OpCapability GenericPointer
+OpCapability Linkage
+OpMemoryModel Physical32 OpenCL
+%i32 = OpTypeInt 32 1
+%u32 = OpTypeInt 32 0
+%f32 = OpTypeFloat 32
+)";
+
+  SpirvStats stats;
+  stats.opcode_markov_hist.resize(2);
+
+  CompileAndAggregateStats(code1, &stats);
+  ASSERT_EQ(2u, stats.opcode_markov_hist.size());
```

Comment:
No change requested.

But I suspect some of this might have been easier to read and write if you had used brace-initialization and ::testing::Eq and its ability to match containers?   In other tests that tends clean up checks against sizes of containers.